plugins/modules/pipx.py

```diff
@@ -120,6 +120,12 @@
             - "B(Warning): C(pipx) documentation states this is an B(experimental) feature subject to change."
         type: str
         version_added: 9.3.0
+    global:
+        description:
+            - The module will pass the C(--global) argument to C(pipx), to execute actions in global scope.
+            - The C(--global) is only available in C(pipx>=1.6.0), make sure to have a compatible version when using this option.
+        type: bool
```

Comment:
Since this option doesn't change the installed package's state, it's better to have a default. That way the user knows that `global=false` is the default.
```suggestion
        type: bool
        default: false
```